package.json

```diff
@@ -19,8 +19,9 @@
     "autoprefixer": "^10.4.20",
     "docsy": "github:google/docsy#semver:0.11.0",
     "hugo-extended": "0.140.2",
-    "netlify-cli": "^17.35.0",
-    "postcss": "^8.4.45",
+    "mkdirp": "^3.0.1",
+    "netlify-cli": "^18.0.1",
+    "postcss": "^8.5.1",
```

Comment:
Should we add a comment so we don't forget to remove the dependency once `google/docsy` addresses the issue? I'm fine either way, to be honest :smiley:

Reply:
Good idea - added a TODO.  Feels a bit strange doing that in a JSON file but apparently it's a thing https://stackoverflow.com/questions/14221579/how-do-i-add-comments-to-package-json-for-npm-install 😂